tests/data/dummy_insertions.pdb

```diff
@@ -68,7 +68,17 @@ ATOM      3  HB2 ALA B   7      36.167  42.973   1.802  1.00  0.00           H
 ATOM      3  HB3 ALA B   7      35.847  44.180   3.181  1.00  0.00           H  
 ATOM      3  C   ALA B   7      36.213  46.067   1.258  1.00  0.00           C  
 ATOM      3  O   ALA B   7      36.287  47.046   2.028  1.00  0.00           O  
-TER       3      ALA B   7
+ATOM      3  N   ALA B   7A     34.725  44.679  -0.087  1.00  0.00           N  
+ATOM      3  H   ALA B   7A     34.813  43.714  -0.371  1.00  0.00           H  
+ATOM      3  CA  ALA B   7A     35.081  45.036   1.305  1.00  0.00           C  
+ATOM      3  HA  ALA B   7A     34.200  45.604   1.876  1.00  0.00           H  
+ATOM      3  CB  ALA B   7A     35.381  43.810   2.140  1.00  0.00           C  
+ATOM      3  HB1 ALA B   7A     34.382  43.229   2.456  1.00  0.00           H  
+ATOM      3  HB2 ALA B   7A     36.167  42.973   1.802  1.00  0.00           H  
+ATOM      3  HB3 ALA B   7A     35.847  44.180   3.181  1.00  0.00           H  
+ATOM      3  C   ALA B   7A     36.213  46.067   1.258  1.00  0.00           C  
+ATOM      3  O   ALA B   7A     36.287  47.046   2.028  1.00  0.00           O  
+TER       3      ALA B   7A
```

Comment:
The test suite needs and additional test that accounts for the following situation:

```
ATOM      3  N   ALA B   7      34.725  44.679  -0.087  1.00  0.00           N·· 
ATOM      3  H   ALA B   7      34.813  43.714  -0.371  1.00  0.00           H·· 
ATOM      3  CA  ALA B   7      35.081  45.036   1.305  1.00  0.00           C·· 
ATOM      3  HA  ALA B   7      34.200  45.604   1.876  1.00  0.00           H·· 
ATOM      3  CB  ALA B   7      35.381  43.810   2.140  1.00  0.00           C·· 
ATOM      3  HB1 ALA B   7      34.382  43.229   2.456  1.00  0.00           H·· 
ATOM      3  HB2 ALA B   7      36.167  42.973   1.802  1.00  0.00           H·· 
ATOM      3  HB3 ALA B   7      35.847  44.180   3.181  1.00  0.00           H·· 
ATOM      3  C   ALA B   7      36.213  46.067   1.258  1.00  0.00           C·· 
ATOM      3  O   ALA B   7      36.287  47.046   2.028  1.00  0.00           O·· 
ATOM      3  N   ALA B   7A     34.725  44.679  -0.087  1.00  0.00           N·· 
ATOM      3  H   ALA B   7A     34.813  43.714  -0.371  1.00  0.00           H·· 
ATOM      3  CA  ALA B   7A     35.081  45.036   1.305  1.00  0.00           C·· 
ATOM      3  HA  ALA B   7A     34.200  45.604   1.876  1.00  0.00           H·· 
ATOM      3  CB  ALA B   7A     35.381  43.810   2.140  1.00  0.00           C·· 
ATOM      3  HB1 ALA B   7A     34.382  43.229   2.456  1.00  0.00           H·· 
ATOM      3  HB2 ALA B   7A     36.167  42.973   1.802  1.00  0.00           H·· 
ATOM      3  HB3 ALA B   7A     35.847  44.180   3.181  1.00  0.00           H·· 
ATOM      3  C   ALA B   7A     36.213  46.067   1.258  1.00  0.00           C·· 
ATOM      3  O   ALA B   7A     36.287  47.046   2.028  1.00  0.00           O·· 
ATOM      3  N   ALA B   8      34.725  44.679  -0.087  1.00  0.00           N·· 
ATOM      3  H   ALA B   8      34.813  43.714  -0.371  1.00  0.00           H·· 
ATOM      3  CA  ALA B   8      35.081  45.036   1.305  1.00  0.00           C·· 
ATOM      3  HA  ALA B   8      34.200  45.604   1.876  1.00  0.00           H·· 
ATOM      3  CB  ALA B   8      35.381  43.810   2.140  1.00  0.00           C·· 
ATOM      3  HB1 ALA B   8      34.382  43.229   2.456  1.00  0.00           H·· 
ATOM      3  HB2 ALA B   8      36.167  42.973   1.802  1.00  0.00           H·· 
ATOM      3  HB3 ALA B   8      35.847  44.180   3.181  1.00  0.00           H·· 
ATOM      3  C   ALA B   8      36.213  46.067   1.258  1.00  0.00           C·· 
ATOM      3  O   ALA B   8      36.287  47.046   2.028  1.00  0.00           O·· 
TER       3      ALA B   8                                                      
```

Reply:
Added this as a different chain to the test file.